testsuite/E30.py

```diff
@@ -118,6 +118,28 @@ def a():
     a()
 #:
 
+#: E306:3:5
```

Comment:
Sorry for the delay. I'm not sure if okay tests apply here, because I just divided a set of already existent error types in two, so non-errors are still the same.